ovnnbimp.go

```diff
@@ -88,9 +94,15 @@ func (odbi *ovnDBImp) oMapContians(s, t map[interface{}]interface{}) bool {
 }
 
 func (odbi *ovnDBImp) getRowUUIDContainsUUID(table, field, uuid string) (string, error) {
-	odbi.cachemutex.Lock()
-	defer odbi.cachemutex.Unlock()
-	for id, drows := range odbi.cache[table] {
+	odbi.cachemutex.RLock()
+	defer odbi.cachemutex.RUnlock()
+
+	cacheTable, ok := odbi.cache[table]
```

Comment:
If table doesn't exist, doesn't it mean there is something fundamentally wrong? E.g. schema problem? Returning ErrorNotFound seems not appropriate for this case.

Reply:
O think that i can create special error for such case